app/javascript/components/specifications-list/SpecList.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import TopNav from '../shared/TopNav';
+import { Link } from 'react-router-dom';
+
+const SpecList = () => {
+  const [specs, setSpecs] = useState([]);
+
+  return (
+    <div className="wrapper">
+      <TopNav />
+      <div className="container-fluid container-wrapper">
+        <div className="row">
+          <div className="col p-lg-5 pt-5">
+            <h4>My Specifications</h4>
+
+            <div className="table-responsive">
+              <table className="table">
+                <thead>
+                  <tr>
+                    <th scope="col">Specification Name</th>
+                    <th scope="col">Version</th>
+                    <th scope="col">Mapped</th>
+                    <th scope="col">Status</th>
+                    <th scope="col">Author</th>
+                    <th scope="col">
+                      <select className="form-control">
+                        <option value="user">Only My Mappings</option>
+                        <option value="all">All Mappings</option>
+                      </select>
+                    </th>
+                  </tr>
+                </thead>
+                <tbody>
+                  { specs.length > 0 ? 
```

Comment:
0 is a falsey value in JS, so `> 0` is redundant.

Reply:
@excelsior it's to make the code self-explanatory. In this case, in my opinion, is easier to understand, by reading the code, that we want to print the list only when the `length > 0` condition is `true`.

I'm happy to change it anyway if you don't agree, but I'll wait for your input before merging.